Dockerfile

```diff
@@ -12,12 +12,13 @@ EXPOSE 8000
 
 USER app
 
+COPY ./requirements.txt /app/requirements.txt
+RUN pip install -r requirements.txt
+
 COPY . /app
 COPY .env-dist /app/.env
 
-RUN pip install -r requirements.txt
-
-RUN mkdir -p /app/static
+RUN mkdir -p /app/static/CACHE
```

Comment:
Hmm ... we may not need this at all, if we `COPY . /app`, then `static/CACHE` should now copy over.